doc/cephfs/fstab.rst

```diff
@@ -2,8 +2,8 @@
  Mount CephFS in your File Systems Table
```

Comment:
I see now that you've already been working on this. I filed https://tracker.ceph.com/issues/42601 for your other doc PR.

While you're here, let's just remove this doc and merge it with the respective fuse or kernel docs. Use a separate section for persistent mounts.